bundles/org.openhab.binding.hue/doc/readme_v2.md

```diff
@@ -0,0 +1,189 @@
+# Philips Hue Binding Configuration for API v2
+
+[Back to Overview](../README.md#philips-hue-binding)
+
+## Supported Things
+
+The binding supports *bridge*, *light*, *button*, and *sensor* devices.
+Lights can be of any type from a simple on/off light, through dimmable monochrome lights, to full colour dimmable lights.
+Buttons are devices that contain one or more push buttons.
+Sensors can be (for example) light level sensors, temperature sensors, or motion sensors.
+
+## Thing Configuration
+
+### Bridge
+
+The Hue Bridge requires the IP address as a configuration value in order for the binding to know where to access it.
+In the thing file, this looks e.g. like
+
+```java
+Bridge hue:bridge:1 [ ipAddress="192.168.0.64" ]
```

Comment:
Currently:
```suggestion
Bridge hue:clip2:1 [ ipAddress="192.168.0.64" ]
```

but mostly a reminder that this needs to be adapted according to outcome of https://github.com/openhab/openhab-addons/pull/13570#discussion_r1148646906.